backend/src/services/interview/interview.service.ts

```diff
@@ -91,6 +94,56 @@ class Interview {
     };
   }
 
+  public async create({
+    categoryId,
+    intervieweeUserId,
+    status,
+  }: InterviewsCreateRequestDto): Promise<InterviewsResponseDto> {
+    const interviewByUserIdAndCategoryId =
+      await this.getInterviewByIntervieweeUserIdAndCategoryId(
+        intervieweeUserId,
+        categoryId,
+      );
+
+    const hasInterview = Boolean(interviewByUserIdAndCategoryId);
+
+    if (
+      hasInterview &&
+      interviewByUserIdAndCategoryId?.status !== InterviewStatus.REJECTED
```

Comment:
what if interview was canceled?

Reply:
Good point. I haven't thought of adding this status. But I think we can Reject the interview, if it needs to be cancelled. @what1s1ove should I add "Cancelled" status or we can Reject the ones, that need to be cancelled?

Reply:
let's make it in another task